src/main/java/pro/cloudnode/smp/nations/commands/NationsCommand.java

```diff
@@ -1,22 +1,422 @@
+                    sendMessage(t(Messages.USAGE, label, "option", "color <color>"));
+                    return;
+                }
+                String color = args[2].toLowerCase();
+                boolean isHex = color.startsWith("#") && color.length() == 7 && color.matches("#[0-9a-fA-F]+");
+                if (!COLORS.contains(color) && !isHex) {
+                    sendMessage(t(Messages.INVALID_COLOR));
+                    return;
+                }
+
+                nation.color = color;
+                sendMessage(t(Messages.COLOR_SET, color));
+                nation.update();
+
+                break;
+            default:
+                sendMessage(t(Messages.USAGE, label, "option", "<key> <value>"));
+                break;
+        }
+    }
+
+    private void info(CommandSender sender, String label, String[] args) {
+        if (args.length > 1) {
+            sendMessage(t(Messages.USAGE, label, "info", ""));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().getPlayerNation(getPlayer().getUniqueId());
+        if (nation == null) {
+            sendMessage(t(Messages.NOT_IN_NATION));
+            return;
+        }
+
+        sendMessage(t(Messages.INFO_HEADER, nation));
+        sendMessage(t(Messages.INFO_LEADER, Bukkit.getPlayer(nation.leader).getName()));
+        sendMessage(t(Messages.INFO_MEMBERS, nation.members.stream().map(Bukkit::getPlayer).filter(Objects::nonNull).map(Player::getName).toList().toString()));
+    }
+
+    private void quit(CommandSender sender, String label, String[] args) {
+        if (args.length > 1) {
+            sendMessage(t(Messages.USAGE, label, "quit", ""));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().getPlayerNation(getPlayer().getUniqueId());
+        if (nation == null) {
+            sendMessage(t(Messages.NOT_IN_NATION));
+            return;
+        }
+
+        if (nation.members.isEmpty() && nation.leader.equals(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.NATION_DISBANDED, nation));
+            Nations.getNationManager().remove(nation);
+            return;
+        }
+
+        if (nation.leader.equals(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.CANT_QUIT_AS_LEADER));
+            return;
+        }
+
+        sendMessage(t(Messages.QUIT_NATION, nation));
+        nation.removeMember(getPlayer().getUniqueId());
+    }
+
+    private void list(CommandSender sender, String label, String[] args) {
+        sendMessage(t(Messages.LIST_HEADER));
+        if (Nations.getNationManager().nations.isEmpty()) {
+            sendMessage(t(Messages.NO_NATIONS));
+            return;
+        }
+        for (Nation nation : Nations.getNationManager().nations.values()) {
+            sendMessage(t(Messages.LIST_ITEM, nation));
+        }
+    }
+
+    private void forceDelete(CommandSender sender, String label, String[] args) {
```

Comment:
i see now why this is `forceDelete`. I think it would be much better as:

nations.delete.other -> `delete <nation>`
nations.delete -> `delete`

and under 1 method